### DISCUSSION TITLE
Option to browse music by folder structure tree view?

### DISCUSSION THREAD

**Discussion by tomasts248** [navidrome/navidrome#2374] [Ideas]
I Apologize if this is already done, I just didn't see it on the demo..
IMHO if it hasn't I think would be a very useful addition.
Further even would be great to be able to recursively share a folder

**Comment by tomasts248** [comment]
Okey at least sharing a folder is planned, but not sure about the initial question. Foobar2000 supports tree like folder navigation for example and for anyone who has spent time organizing the folders this is a must. Okey found a related thread 
https://github.com/navidrome/navidrome/issues/739
This feature is not planned :/

**Comment by deluan** [comment]
> This feature is not planned :/

Yes. For reference: https://www.navidrome.org/docs/faq/#-can-you-add-a-browsing-by-folder-optionmode-to-navidrome

**Reply by tomasts248** [reply]
Thanks!
